src/test/java/edu/hm/hafner/analysis/Assertions/IssueAssert.java

```diff
@@ -0,0 +1,238 @@
+package edu.hm.hafner.analysis.Assertions;
+
+import java.util.Objects;
+import java.util.UUID;
+
+import org.assertj.core.api.AbstractAssert;
+
+import edu.hm.hafner.analysis.Issue;
+import edu.hm.hafner.analysis.Priority;
+
+/**
+ * Custom assertion for issue.
+ * @author Tom Maier
+ * @author Johannes Arzt
+ */
+public class IssueAssert extends AbstractAssert<IssueAssert, Issue> {
+
+    /**
+     * Custom constructor.
+     *
+     * @param actual current issue
+     */
+    public IssueAssert(Issue actual) {
+        super(actual, IssueAssert.class);
+    }
+
+    /**
+     * Custom assert function for issue.
+     *
+     * @param actual current issue
+     * @return custom assertion
+     */
+    public static IssueAssert assertThat(Issue actual) {
+        return new IssueAssert(actual);
+    }
+
+    /**
+     * Check issue for specific file name.
+     *
+     * @param fileName filter for file name
+     * @return custom assertion
+     */
+    public IssueAssert hasFileName(String fileName) {
+        isNotNull();
+
+        if (!Objects.equals(actual.getFileName(), fileName)) {
+            failWithMessage("Expected file Name to be <%s> but was <%s>", fileName, actual.getFileName());
```

Comment:
Der Standard ist:
`private static final String EXPECTED_BUT_WAS_MESSAGE = "%nExpecting:%n <%s>%nto be equal to:%n <%s>%nbut was not.";`